roles/infra/tasks/main.yml

```diff
@@ -46,6 +46,12 @@
     content: "{{ _terraform_backend_config | to_json }}"
     dest: "{{ terraform_project_path }}/backend_config.json"
 
+- name: Set terraform to rebuild patched appliances
+  include_tasks: rebuild.yml
+  when:
+    - terraform_rebuild_nodes | default(false)
```

Comment:
@assumptionsandg 

I meant set a default in `roles/infra/defaults/main.yml` rather than this. Can we just change that? 🙂

Reply:
Done